cli/src/config/misc.toml

```diff
@@ -13,7 +13,7 @@ allow-filesets = false
 always-allow-large-revsets = false
 diff-instructions = true
 paginate = "auto"
-pager = { command = ["less", "-FRX"], env = { LESSCHARSET = "utf-8" } }
+pager = { command = ["less"], env = { LESS = "-FRX", LESSCHARSET = "utf-8" } }
```

Comment:
Perhaps we could have a more generic `env_default = { ... }` that's like `env`, but does not override environment variables that are already set?

Reply:
Sounds reasonable. I can add that.

Reply:
Done.

Reply:
Thanks! I added some optional suggestions. I haven't realized how much work I was asking you to do, and there are still some confusing issues left, but I think this is fine as is.